docs/audit-manager-developer-guide.md

```diff
@@ -0,0 +1,124 @@
+# Audit Manager Developers' Guide
+
+## Overview
+[Audit-Manager](https://docs.mosip.io/1.2.0/modules/audit-manager) module provides audit related functionalities.
+
+
+Below is a list of tools required in Commons:
```

Comment:
Below is a list of tools required in Commons:    -- Should this be audit manager or commons?